app/assets/javascripts/angular/components/publicSurvey/publicSurvey.html

```diff
@@ -25,7 +25,8 @@
                 <div class="pa4 tl">
                     <h1 class="pivot-gray-darker f2 mt3 normal mb0">{{ $ctrl.survey.title }}</h1>
                     <p class="lh-copy mt1" ng-if="!$ctrl.thankYou">{{ $ctrl.survey.login_paragraph }}</p>
-                    <p class="lh-copy mt1" ng-if="$ctrl.thankYou">{{ $ctrl.survey.post_survey_message }}</p>
+                    <p class="lh-copy mt1" ng-if="$ctrl.thankYou && $ctrl.survey.post_survey_message">{{ $ctrl.survey.post_survey_message }}</p>
+                    <p class="lh-copy mt1" ng-if="$ctrl.thankYou && !$ctrl.survey.post_survey_message">{{ 'surveys:publicView.post_survey_message' | t }}</p>
```

Comment:
Would just having one of these be cleaner? Hopefully my Angular template syntax works... Maybe I'm expecting too much from Angular...
```suggestion
                    <p class="lh-copy mt1" ng-if="$ctrl.thankYou">{{ $ctrl.survey.post_survey_message || ('surveys:publicView.post_survey_message' | t) }}</p>
```

Reply:
It 'should' have worked. It seems the il8next library is finicky, ran into this weirdness earlier

Reply:
Huh, weird. Whatever. Does it throw errors or just not render anything?

Reply:
renders nothing